packages/builder/src/components/userInterface/ComponentsHierarchyChildren.svelte

```diff
@@ -55,8 +140,27 @@
           components={component._children}
           {currentComponent}
           {onSelect}
+          {dragDropStore}
           level={level + 1} />
       {/if}
+
+      {#if $dragDropStore && $dragDropStore.targetComponent === component && $dragDropStore.dropPosition == 'inside'}
+        <div
+          on:drop={drop}
+          ondragover="return false"
+          ondragenter="return false"
+          class="budibase__nav-item item drop-item"
+          style="margin-left: {(level + 2) * 20 + 40}px" />
+      {/if}
+
+      {#if $dragDropStore && $dragDropStore.targetComponent === component && $dragDropStore.dropPosition == 'below'}
+        <div
+          on:drop={drop}
+          ondragover="return false"
+          ondragenter="return false"
+          class="budibase__nav-item item drop-item"
+          style="margin-left: {level * 20 + 40}px" />
```

Comment:
NAB: Could be simplified to only conditionally change the `style` prop, since that's the only difference between these 2 conditions